environemnt set up
Running the batch script
2020-12-19 12:18:37.253162: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:18:45.751450: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-19 12:18:45.768771: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2400000000 Hz
2020-12-19 12:18:45.772794: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56209619aac0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-19 12:18:45.772828: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-19 12:18:45.777822: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-19 12:18:45.954017: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56209619d2c0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-12-19 12:18:45.954098: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Tesla V100-SXM2-32GB, Compute Capability 7.0
2020-12-19 12:18:45.960882: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:18:00.0 name: Tesla V100-SXM2-32GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 836.37GiB/s
2020-12-19 12:18:45.960960: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:18:46.036295: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:18:46.076859: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:18:46.091702: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:18:46.156136: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:18:46.174853: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:18:46.332554: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:18:46.336227: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:18:46.337058: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:18:48.763274: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:18:48.763345: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:18:48.763360: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:18:48.769442: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100-SXM2-32GB, pci bus id: 0000:18:00.0, compute capability: 7.0)
2020-12-19 12:18:48.776700: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:18:00.0 name: Tesla V100-SXM2-32GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 836.37GiB/s
2020-12-19 12:18:48.776779: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:18:48.776829: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:18:48.776870: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:18:48.776903: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:18:48.776936: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:18:48.776967: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:18:48.777000: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:18:48.785835: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:18:48.785889: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:18:48.785907: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:18:48.785924: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:18:48.794167: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100-SXM2-32GB, pci bus id: 0000:18:00.0, compute capability: 7.0)
2020-12-19 12:18:49.547456: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:18:00.0 name: Tesla V100-SXM2-32GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 836.37GiB/s
2020-12-19 12:18:49.547531: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:18:49.547567: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:18:49.547588: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:18:49.547607: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:18:49.547626: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:18:49.547652: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:18:49.547672: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:18:49.553229: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:18:49.556467: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:18:00.0 name: Tesla V100-SXM2-32GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 836.37GiB/s
2020-12-19 12:18:49.556507: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:18:49.556532: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:18:49.556550: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:18:49.556568: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:18:49.556585: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:18:49.556602: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:18:49.556620: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:18:49.561861: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:18:49.561904: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:18:49.561915: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:18:49.561926: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:18:49.566753: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100-SXM2-32GB, pci bus id: 0000:18:00.0, compute capability: 7.0)
Using TensorFlow backend.
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x2b1d39153400> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Does the system is built with CUDA?: True
Default GPU Device: /device:GPU:0
====> Start running baseline.py
Classes: ['Tomato___Late_blight', 'Tomato___Tomato_Yellow_Leaf_Curl_Virus', 'Peach___healthy', 'Grape___Esca_(Black_Measles)', 'Potato___Late_blight', 'Pepper,_bell___Bacterial_spot', 'Strawberry___healthy', 'Orange___Haunglongbing_(Citrus_greening)', 'Tomato___Leaf_Mold', 'Apple___Black_rot', 'Strawberry___Leaf_scorch', 'Tomato___Early_blight', 'Cherry_(including_sour)___healthy', 'Corn_(maize)___Common_rust_', 'Blueberry___healthy', 'Potato___Early_blight', 'Pepper,_bell___healthy', 'Apple___Cedar_apple_rust', 'Grape___Leaf_blight_(Isariopsis_Leaf_Spot)', 'Tomato___Tomato_mosaic_virus', 'Tomato___Target_Spot', 'Tomato___healthy', 'Peach___Bacterial_spot', 'Corn_(maize)___Cercospora_leaf_spot Gray_leaf_spot', 'Tomato___Spider_mites Two-spotted_spider_mite', 'Corn_(maize)___healthy', 'Squash___Powdery_mildew', 'Cherry_(including_sour)___Powdery_mildew', 'Tomato___Bacterial_spot', 'Grape___Black_rot', 'Apple___healthy', 'Potato___healthy', 'Corn_(maize)___Northern_Leaf_Blight', 'Tomato___Septoria_leaf_spot', 'Raspberry___healthy', 'Soybean___healthy', 'Apple___Apple_scab', 'Grape___healthy']
Number of classes: 38
Total number of images: 54305
first_folder_path: /fs/scratch/PAA0023/dong760/PlantVillage-Dataset/raw/color/Tomato___Late_blight
Number of folder: 38
sample image path: /fs/scratch/PAA0023/dong760/PlantVillage-Dataset/raw/color/Tomato___Late_blight/5fdf3288-d5f4-4c70-9b12-95222b82d415___RS_Late.B 4942.JPG
Image size: (256, 256, 3)
Found 43456 images belonging to 38 classes.
Found 10849 images belonging to 38 classes.
Train size: 43456

====> Statistics: epochs=5, batch_size=1024, validation_split=0.2, lr=0.001, momentum=0.9, steps_per_epoch =100, feature shape= (256, 256, 3), no_classes=38, loss_function=categorical_crossentropy
Epoch 1/5
2020-12-19 12:19:23.897783: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:19:45.277054: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
WARNING:tensorflow:AutoGraph could not transform <function Model.make_test_function.<locals>.test_function at 0x2b1d3ac2c1e0> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
43/43 - 1176s - loss: 2.2070 - accuracy: 0.4770 - precision: 0.9730 - recall: 0.1815 - auc: 0.8933 - val_loss: 1.2360 - val_accuracy: 0.7300 - val_precision: 0.9708 - val_recall: 0.4112 - val_auc: 0.9800
Epoch 2/5
43/43 - 1110s - loss: 0.8997 - accuracy: 0.8092 - precision: 0.9780 - recall: 0.5570 - auc: 0.9907 - val_loss: 0.6944 - val_accuracy: 0.8561 - val_precision: 0.9726 - val_recall: 0.6512 - val_auc: 0.9951
Epoch 3/5
43/43 - 1097s - loss: 0.5680 - accuracy: 0.8886 - precision: 0.9788 - recall: 0.7200 - auc: 0.9971 - val_loss: 0.5088 - val_accuracy: 0.8931 - val_precision: 0.9746 - val_recall: 0.7591 - val_auc: 0.9975
Epoch 4/5
43/43 - 1094s - loss: 0.4294 - accuracy: 0.9171 - precision: 0.9799 - recall: 0.7976 - auc: 0.9985 - val_loss: 0.4116 - val_accuracy: 0.9134 - val_precision: 0.9749 - val_recall: 0.8052 - val_auc: 0.9985
Epoch 5/5
43/43 - 1102s - loss: 0.3482 - accuracy: 0.9349 - precision: 0.9821 - recall: 0.8447 - auc: 0.9990 - val_loss: 0.3554 - val_accuracy: 0.9261 - val_precision: 0.9770 - val_recall: 0.8366 - val_auc: 0.9990
WARNING:tensorflow:From /users/PAA0023/dong760/miniconda3/envs/tf_latest/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2020-12-19 13:54:25.820346: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /users/PAA0023/dong760/miniconda3/envs/tf_latest/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
WARNING:tensorflow:AutoGraph could not transform <function canonicalize_signatures.<locals>.signature_wrapper at 0x2b1db91d7a60> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
{'loss': [2.206979513168335, 0.8996766209602356, 0.5679932832717896, 0.42941856384277344, 0.3482460081577301], 'accuracy': [0.4769652187824249, 0.8092092871665955, 0.8885769248008728, 0.9171115756034851, 0.9349226951599121], 'precision': [0.9729896187782288, 0.978020191192627, 0.9787899851799011, 0.9798722267150879, 0.9821280241012573], 'recall': [0.18153995275497437, 0.5570231676101685, 0.7199926376342773, 0.7976343631744385, 0.8447394967079163], 'auc': [0.893276035785675, 0.9906573295593262, 0.9970954656600952, 0.9984593391418457, 0.999049961566925], 'val_loss': [1.2359929084777832, 0.6943528056144714, 0.5088353753089905, 0.41158491373062134, 0.35540732741355896], 'val_accuracy': [0.7300211787223816, 0.8561157584190369, 0.8930777311325073, 0.9133560657501221, 0.9260761141777039], 'val_precision': [0.9708378911018372, 0.9726046323776245, 0.9745562076568604, 0.974891185760498, 0.9769644737243652], 'val_recall': [0.4111899733543396, 0.6512120962142944, 0.7590561509132385, 0.8052355051040649, 0.8365747928619385], 'val_auc': [0.98004150390625, 0.9950945973396301, 0.997529923915863, 0.9985320568084717, 0.9989625215530396]}
date and time = 19-12-2020_13-54-18
Saving the model to path: /users/PAA0023/dong760/plant_leaves_diagnosis/saved_models/InceptionV3_BatchSize_1024_0.2ValSplit_19-12-2020_13-54-18


p0353.ten.osc.edu:
                                                                               Req'd  Req'd   Elap 
Job id               Username Queue    Name                 SessID NDS   TSK   Memory Time Use S Time 
-------------------- -------- -------- -------------------- ------ ----- ----- ------ ----- - -----
2564371              dong760  gpuseria ondemand/sys/dashboa --         1    28     -- 05:00 R 04:04
2565109              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:32
2565110              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:33
2565113              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:32
2565115              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:32
2565116              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:36
2565118              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:35
2565119              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 C 01:33
The date when running current script is :
Sat Dec 19 13:54:46 EST 2020
